test/Microsoft.IdentityModel.Tokens.Saml.Tests/Saml2SecurityTokenHandlerTests.cs

```diff
@@ -690,10 +690,9 @@ public void ValidateToken(Saml2TheoryData theoryData)
         {
             TestUtilities.WriteHeader($"{this}.ValidateToken", theoryData);
             var context = new CompareContext($"{this}.ValidateToken, {theoryData}");
-            ClaimsPrincipal retVal = null;
             try
             {
-                retVal = theoryData.Handler.ValidateToken(theoryData.Token, theoryData.ValidationParameters, out SecurityToken validatedToken);
+                var retVal = theoryData.Handler.ValidateToken(theoryData.Token, theoryData.ValidationParameters, out SecurityToken validatedToken);
```

Comment:
Can we just call handler.ValidateToken() without storing the return value in a variable? (also relevant in the SamlSecurityTokenHandlerTests)

Reply:
Thanks, you are right. I removed a few redundant initialization, but sometimes when didn't need to initialize at all :)

Reply:
addressed with: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1158/commits/9e94ffea506bcab20e4cede543525783e149715c.